examples/document_conversion_integration.v1-experimental.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+
+var watson = require('watson-developer-cloud');
+var async  = require('async');
+var fs = require('fs');
+
+var retrieve = watson.retrieve_and_rank({
+  username: 'INSERT YOUR USERNAME FOR THE SERVICE HERE',
+  password: 'INSERT YOUR PASSWORD FOR THE SERVICE HERE',
+  version: 'v1',
+  url: 'https://gateway.watsonplatform.net/retrieve-and-rank/api'
+});
+
+var document_conversion = watson.document_conversion({
+  username: 'INSERT YOUR USERNAME FOR THE SERVICE HERE',
+  password: 'INSERT YOUR PASSWORD FOR THE SERVICE HERE',
+  version: 'v1-experimental'
+});
+
```

Comment:
probably add a comment saying that the username and password will be different for each service and that users can't use their bluemix username and password